common/interpreter/wawaka_wasm/README.md

```diff
@@ -89,6 +89,37 @@ pdo-test-contract --no-ledger --interpreter wawaka --contract mock-contract \
     --expressions ./mock-contract/test-long.exp --loglevel info
 ```
 
+## Ahead-of-Time Compiled WASM
+** Disclaimer: Support for this feature is under active development and will be rolled-out at a later date. AoT mode cannot currently be enabled for testing. **
+
+### Setup
+
+By default, wawaka will be built for interpreted wasm contracts. If you would like to enable
+ahead-of-time (AoT) compiled wasm contracts, set the environment variable `WASM_MODE` (default: `INTERP`):
+
+```bash
+export WASM_MODE=AOT
+```
+
+With AoT wasm enabled, the `wamrc` compiler (including dependencies)
+shipped as part of the WAMR codebase will be built
+as part of the PDO build.
+
+### Building and running wawaka Contracts
+
+We provide a new cmake function `BUILD_AOT_CONTRACT` that
+automatically invokes `wamrc` on a given wasm bytecode file,
+and generates the corresponding `.aot` binary file. 
+To build an AoT-compiled wawaka contract, add the following call to the contract's
+`CMakeList.txt`:
+
+```
```

Comment:
Maybe worth adding a test which illustrates that and as well acts as a corresponding unit-test?

Reply:
@cmickeyb and I had discussed that we would wait to have all the code provenance pieces in place before we publish any actual AoT-based code, which is why I didn't include any actual tests in this PR.